### DISCUSSION TITLE
Configuration

### DISCUSSION THREAD

**Discussion by kubitre** [ory/hydra#2730] [Ideas]
Hello! I maintainer of library gostructor(https://github.com/goreflect/gostructor). Maybe my library will be useful to you? It is quite convenient to configure the application, and you do not need to write a lot of boilerplate code. If u have example usages i can set into this tred. 

Also, if you have thoughts that my library is missing, you can write to me

**Comment by aeneasr** [comment]
Hello there! Thank you for the idea - the library and logo look great :) Right now we are very happy with Koanf and the tooling we have written around it so it is unlikely for us to move on that front in the near future. But I'll definitely check out the project and maybe we find some use in another product! :)
